.github/workflows/maven.yml

```diff
@@ -0,0 +1,17 @@
+name: CoffeeNet Maven Build
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
```

Comment:
In this or in next commits/prs we should also use https://github.com/actions/cache

Reply:
done.